tracks/swift/exercises/leap/mentoring.md

```diff
@@ -9,21 +9,24 @@ struct Year {
     let isLeapYear: Bool
```

Comment:
Update reasonable solutions heading to level 3, and remove the Leap title header.